examples/mocks/index.ts

```diff
@@ -10,16 +10,13 @@ interface LoginResponse {
 }
 
 export const handlers = [
-  rest.post<LoginBody, LoginResponse>('/login', (req, res, ctx) => {
-    const { username } = req.body
-    return res(
-      ctx.json({
-        username,
-        firstName: 'John',
-      }),
-    )
+  http.post<LoginBody, LoginResponse>('/login', async ({ request }) => {
+    const user = await request.json()
+    const { username } = user
+
+    return new Response(JSON.stringify({ username, firstName: 'John' }))
```

Comment:
Let's use `HttpResponse.json` instead? You can import it from `msw`. Then you can drop `JSON.stringify()` here. 

- https://mswjs.io/docs/api/http-response/